src/Sylius/Bundle/CoreBundle/Form/Type/Product/ChannelPricingType.php

```diff
@@ -35,7 +36,12 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 $form = $event->getForm();
 
                 $form->add('price', MoneyType::class, [
-                    'label' => $channel->getName(),
+                    'label' => 'Price for ' . $channel->getName(),
```

Comment:
Maybe this can help: 
http://stackoverflow.com/questions/19593851/symfony2-form-use-translator-for-attr-attribute

Although, it seems this feature is not provided out of the box by symfony: https://github.com/symfony/symfony/pull/10125#issuecomment-112468612